substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrGlobalMemory.java

```diff
@@ -73,11 +73,13 @@ public void teardown() {
         }
     }
 
+    @Uninterruptible(reason = "May be called by uninterruptible code", mayBeInlined = true)
```

Comment:
Use `@Uninterruptible(reason = "Called from uninterruptible code.", mayBeInlined = true)` instead (we use that specific wording for the reason throughout the project). Same applies to the method below.